articles/synapse-analytics/known-issues.md

```diff
@@ -101,6 +102,12 @@ When making a change to the [tags](../azure-resource-manager/management/tag-reso
 
 **Workaround**: You can confirm that the change to the tags was successful and ignore/suppress the error message as needed.
 
+### Proxied Connections may result in failure due to Gateway
+
+When establishing a connection from outside of the Azure network boundary, all connections are proxied through the Gateway as per the Default [Connection policy](security/connectivity-settings.md) for Synapse Workspaces. The same is applied when utilizing Private Endpoints in Synapse workspaces. Due to this policy this can lead to increased latency and reduced throughput when communicating wth the dedicated pool and be affected by Gateway outages.
+
+**Workaround**: Currently there is no workaround for scenario however, development efforts are underway to address this in a future networking update. 
```

Comment:
@chopsakie @charithcaldera This needs to be re-worked. We cannot publish future product plans or promises. For more information, see [What can I publish on learn.microsoft.com](https://learn.microsoft.com/en-us/help/platform/contribute-get-started-channel-guidance)

#hold-off

Reply:
I just removed the future release part.

Reply:
#Sign-Off